cnf-certification-test/accesscontrol/suite.go

```diff
@@ -185,16 +205,55 @@ var _ = ginkgo.Describe(common.AccessControlTestKey, func() {
 	})
 })
 
-// TestSecConCapabilities verifies that non compliant capabilities are not present
-func TestSecConCapabilities(env *provider.TestEnvironment) {
+func TestSysAdminCapability(env *provider.TestEnvironment) {
 	var badContainers []string
 	for _, cut := range env.Containers {
 		if cut.SecurityContext != nil && cut.SecurityContext.Capabilities != nil {
-			for _, ncc := range nonCompliantCapabilities {
-				if strings.Contains(cut.SecurityContext.Capabilities.String(), ncc) {
-					tnf.ClaimFilePrintf("Non compliant %s capability detected in container %s. All container caps: %s", ncc, cut.String(), cut.SecurityContext.Capabilities.String())
-					badContainers = append(badContainers, cut.String())
-				}
+			if strings.Contains(cut.SecurityContext.Capabilities.String(), "SYS_ADMIN") {
+				tnf.ClaimFilePrintf("Non compliant %s capability detected in container %s. All container caps: %s", "SYS_ADMIN", cut.String(), cut.SecurityContext.Capabilities.String())
```

Comment:
I know, I'm not popular with 79 symbols limit per line, but this line is 178 symbols long. @greyerof do you have monitor big enough? :-)

Reply:
`go fmt` doesn't like when I split into multiple lines.  Can be addressed later if needed.

Reply:
Yes, @rdavid I can see the whole line correctly, but I would have put the arguments in the next line. By the way, when showing code in PRs, github uses 8 spaces for tabs by default, but you can change it to whatever number you want (I changed that to 4). It's on settings/appearence:
![image](https://user-images.githubusercontent.com/87083379/208840433-53d5f7e4-6ed6-4675-bf13-5076555bf3cf.png)